.github/workflows/test.yml

```diff
@@ -142,6 +142,7 @@ jobs:
           grafana-app-sdk project init "test-project"
           rm -rf kinds/manifest.cue
           printf "\nreplace github.com/grafana/grafana-app-sdk => $(readlink -f ../grafana-app-sdk)\n" >> go.mod
+          printf "\nreplace github.com/grafana/grafana-app-sdk/logging => $(readlink -f ../grafana-app-sdk/logging)\n" >> go.mod
```

Comment:
Do we need this line when there's a replace of the logging in the gomod?

Reply:
Yes, unfortunately. We will remove it once merged, however!

Reply:
Sorry, I completely skipped over this was already discussed in the thread 😶‍🌫️